grafana-plugin/src/models/channel_filter.ts

```diff
@@ -4,7 +4,6 @@ import { TelegramChannel } from 'models/telegram_channel/telegram_channel.types'
 
 export interface ChannelFilter {
   id: string;
-  order: number;
```

Comment:
Plugin frontend doesn't use `order` anywhere (it uses step indices and `move_to_position` action instead)